### PR TITLE
Cleanup ETT C4 models

### PR DESCRIPTION
**Pull request by tnickelsen**

### CODE REVIEW

File: docs/diagrams/c4-model/dh-base-model.dsl [hunk 1]
```diff
@@ -73,6 +73,13 @@ workspace "DataHub" {
                 tags "Out of focus"
             }
         }
+        group "Microsoft" {
```
Comment (Sondergaard) [inline docs/diagrams/c4-model/dh-base-model.dsl:76]: Why is this group added? It's not being referenced in the changes? The identifier `github` is already used, and is the reason why the pipeline is unable to build.

**Review by Sondergaard — CHANGES_REQUESTED** [review]
We must change the identifier or delete the block entirely if it's not to be used.

File: docs/diagrams/c4-model/dh-base-model.dsl (hunk 1 above)
Comment (tnickelsen) [reply docs/diagrams/c4-model/dh-base-model.dsl:76]: Name has been corrected - I need it to make a ref to Azure AD B2C

File: docs/diagrams/c4-model/dh-base-model.dsl [hunk 2]
```diff
@@ -209,19 +217,19 @@ workspace "DataHub" {
...
-        energyOrigin -> cvr "Reads CVR data" "https"
+        # Energy Track and Trace DK
+        ettDk -> dh2 "Requests measurements" "https"
+        ettDk -> poRegistry "Links to guarantees of origin" "https"
+        ettDk -> poWallet "Places certificates in" "https"
+        ettDk -> poStamp "Issues certificates" "https"
+        ettDk -> mitId "Authenticate users" "https"
+        ettDk -> azureAdB2c "Authenticates using" "OIDC"
+        ettDkUser -> ett "Reads/manages granular certificates" "browser"
+        ettDkThirdPartySystem -> ett "Integrates with platform on behalf of users" "https"
```
Comment (Sondergaard) [inline docs/diagrams/c4-model/dh-base-model.dsl:228]: If you change these lines the diagram will render.

```suggestion
        ettDkUser -> ettDk "Reads/manages granular certificates" "browser"
        ettDkThirdPartySystem -> ettDk "Integrates with platform on behalf of users" "https"
```

**Review by Sondergaard — APPROVED** [review]

**Pull request merged**